Modules/_ssl.c

```diff
@@ -5552,7 +5538,14 @@ _ssl_enum_certificates_impl(PyObject *module, const char *store_name)
         return PyErr_SetFromWindowsErr(GetLastError());
     }
 
-    return result;
+    /* convert set to list */
+    if (result == NULL) {
+        return NULL;
+    } else {
```

Comment:
You could remove this `else` statement (also in the corresponding code in `_ssl_enum_crls_impl`).